docs/facility_e4s.md

```diff
@@ -1,15 +1,15 @@
-Perlmutter | NERSC| 21.11 | 50                  | `gcc@11.2.0`, `nvhpc@21.11`                                                                                | [https://github.com/spack/spack/tree/e4s-21.11](https://github.com/spack/spack/tree/e4s-21.11)                                      | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/spack.yaml](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/spack.yaml)             | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/e4s-21.11.txt](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/e4s-21.11.txt)
-Perlmutter | NERSC| 22.05 | 63                  | `gcc@11.2.0`, `cce@13.0.2`                                                                                | [https://github.com/spack/spack/tree/e4s-22.05](https://github.com/spack/spack/tree/e4s-22.05)                                      | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/spack.yaml](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/spack.yaml)             | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/e4s-22.05.txt](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/e4s-22.05.txt)
+Perlmutter | NERSC | 21.11 | 38 | `gcc@11.2.0`               | [https://github.com/spack/spack/tree/e4s-21.11](https://github.com/spack/spack/tree/e4s-21.11)                                      | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/spack.yaml](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/spack.yaml)             | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/e4s-21.11.txt](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-21.11/e4s-21.11.txt)
+Perlmutter | NERSC | 22.05 | 63 | `gcc@11.2.0`, `cce@13.0.2` | [https://github.com/spack/spack/tree/e4s-22.05](https://github.com/spack/spack/tree/e4s-22.05)                                      | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/spack.yaml](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/spack.yaml)             | [https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/e4s-22.05.txt](https://github.com/spack/spack-configs/blob/main/NERSC/perlmutter/e4s-22.05/e4s-22.05.txt)
```

Comment:
can you please remove line 7. I already have Perlmutter E4S/22.05 entry and it looks incorrect. Its not 63 packages its 71.

Reply:
Strange, this seems like it should have been a merge conflict. --Anyways, now its fixed.

Reply:
thank you this looks good